tests/components/conftest.py

```diff
@@ -51,6 +54,9 @@
     from .sensor.common import MockSensor
     from .switch.common import MockSwitch
 
+# Regex for accessing the integration name from the test path
+RE_REQUEST_DOMAIN = re.compile(r".*tests\/components\/([a-zA-Z_]*)\/.*")
```

Comment:
```suggestion
RE_REQUEST_DOMAIN = re.compile(r".*tests\/components\/([^/]+)\/.*")
```

Why not using `([^/]+)`